main.tf

```diff
@@ -56,4 +58,28 @@ resource "azurerm_role_assignment" "role_assignments" {
   scope                = data.azurerm_subscription.subscription.id
   role_definition_name = local.roles_to_assign[count.index]
   principal_id         = azuread_service_principal.bridgecrew_sp.object_id
+}
+
+resource null_resource "notify_bridgecrew" {
+  triggers = {
+    version = local.version
+  }
+
+  provisioner "local-exec" {
+    command = <<CURL
+      curl --request PUT 'https://www.bridgecrew.cloud/api/v1/integrations/csp' \
```

Comment:
How can I handle it in dev? The url is only for prod.
How can I connect azure to my dev env ?

Reply:
You need to run it locally + change the URL to your dev env

Reply:
In the repo dir, create this file:

provider "azurerm" {
  subscription_id = "SUB_ID"
  features {}
}

module "bridgecrew-read" {
  source = "../"
  org_name = "NAME"
  bridgecrew_token = "TOKEN"
}

output "bridgecrew_output" {
  value = module.bridgecrew-read.bridgecrew_output
}

and run it(terraform apply)

Reply:
why not put is a var? instead of change it locally (can be pushed with wrong url)

Reply:
@guy-suli added base_url variable